src/services/kuma-api/Api.ts

```diff
@@ -8,13 +8,10 @@ type Env = (
   >
 ) => string
```

Comment:
**Suggestion**: The `Env` type can be changed here, too

Reply:
Kinda getting away from the PR here, but I changed this also